tests/test_namespace.py

```diff
@@ -312,10 +313,37 @@ def test_namespace_str(self):
                          "<city: http://www.osp-core.com/city#>")
 
     def test_get_default_rel(self):
+        # default rel defined as flag in entity name
         self.installer.install("city")
         namespace = self.namespace_registry.city
         self.assertEqual(namespace.get_default_rel().name, "hasPart")
 
+        onto_def_rel = os.path.join(
```

Comment:
You're technically not testing the namespace or the get_default method, as the method name or the file name of this test might suggest.

But I guess that's fine. What do you think?

Reply:
I think you are right, there is no need to install the namespace, i can rather directly test the `YmlParser.parse()`. I will put the test somewhere else.

Reply:
Actually, I just found that it is ok. What I want to test is that the installation of the yml will result in the right default relationship in the namespace. I think the confusion is that the `YmlParser` is used for installation and this is where I put my code. So the question is, is this test part of the installation, the namespace or the parser? I could change the name of the test though.

Reply:
Let it as it is. I think it is fine